service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseControllerTest.java

```diff
@@ -59,129 +57,75 @@ void shouldThrowExceptionWhenMigrationNotMappedForMigrationID() {
-                NoticeOfChangeChildAnswersData.builder()
-                    .noticeOfChangeChildAnswers0(NoticeOfChangeAnswers.builder().build())
-                    .noticeOfChangeChildAnswers1(NoticeOfChangeAnswers.builder().build())
-                    .noticeOfChangeChildAnswers2(NoticeOfChangeAnswers.builder().build())
-                    .noticeOfChangeChildAnswers3(NoticeOfChangeAnswers.builder().build())
-                    .noticeOfChangeChildAnswers4(NoticeOfChangeAnswers.builder().build())
-                    .noticeOfChangeChildAnswers5(NoticeOfChangeAnswers.builder().build())
-                    .noticeOfChangeChildAnswers6(NoticeOfChangeAnswers.builder().build())
-                    .noticeOfChangeChildAnswers7(NoticeOfChangeAnswers.builder().build())
-                    .noticeOfChangeChildAnswers8(NoticeOfChangeAnswers.builder().build())
-                    .noticeOfChangeChildAnswers9(NoticeOfChangeAnswers.builder().build())
-                    .noticeOfChangeChildAnswers10(NoticeOfChangeAnswers.builder().build())
-                    .noticeOfChangeChildAnswers11(NoticeOfChangeAnswers.builder().build())
-                    .noticeOfChangeChildAnswers12(NoticeOfChangeAnswers.builder().build())
-                    .noticeOfChangeChildAnswers13(NoticeOfChangeAnswers.builder().build())
-                    .noticeOfChangeChildAnswers14(NoticeOfChangeAnswers.builder().build())
-                    .build()
-            );
-
-            assertThat(responseData.getChildPolicyData()).isEqualTo(
-                ChildPolicyData.builder()
-                    .childPolicy0(blankPolicyFor(SolicitorRole.CHILDSOLICITORA))
-                    .childPolicy1(blankPolicyFor(SolicitorRole.CHILDSOLICITORB))
-                    .childPolicy2(blankPolicyFor(SolicitorRole.CHILDSOLICITORC))
-                    .childPolicy3(blankPolicyFor(SolicitorRole.CHILDSOLICITORD))
-                    .childPolicy4(blankPolicyFor(SolicitorRole.CHILDSOLICITORE))
-                    .childPolicy5(blankPolicyFor(SolicitorRole.CHILDSOLICITORF))
-                    .childPolicy6(blankPolicyFor(SolicitorRole.CHILDSOLICITORG))
-                    .childPolicy7(blankPolicyFor(SolicitorRole.CHILDSOLICITORH))
-                    .childPolicy8(blankPolicyFor(SolicitorRole.CHILDSOLICITORI))
-                    .childPolicy9(blankPolicyFor(SolicitorRole.CHILDSOLICITORJ))
-                    .childPolicy10(blankPolicyFor(SolicitorRole.CHILDSOLICITORK))
-                    .childPolicy11(blankPolicyFor(SolicitorRole.CHILDSOLICITORL))
-                    .childPolicy12(blankPolicyFor(SolicitorRole.CHILDSOLICITORM))
-                    .childPolicy13(blankPolicyFor(SolicitorRole.CHILDSOLICITORN))
-                    .childPolicy14(blankPolicyFor(SolicitorRole.CHILDSOLICITORO))
-                    .build()
-            );
+            assertThat(responseData.getOtherCourtAdminDocuments().size()).isEqualTo(1);
```

Comment:
there is a useful assertion method that you can use here
```java
assertThat(responseData.getOtherCourtAdminDocuments()).hasSize(1);
```
you can then chain this with something like
```java
assertThat(responseData.getOtherCourtAdminDocuments()).hasSize(1)
    .first()
    .extracting(doc -> doc.getValue().getDocumentTitle())
    .isEqualTo("court-document1");
```

Reply:
Changed as suggested